styles/styles.css

```diff
@@ -0,0 +1,389 @@
+* {box-sizing: border-box;}
+
+* {font-family: Georgia, serif;}
+
+
+header {
+  display: flex;
+
+  flex-direction: column;
+  padding: 2em;
+
+  background: rgb(115,136,153);
+  background: linear-gradient(0deg, rgba(115,136,153,1) 0%, rgba(9,60,102,1) 100%);
+  background-size: auto;
+  background-image: url('../images/header-image.jpg');
+
+
+
+}
+
+
+.logo {
+  width: 70%;
+}
+
+.logo img {
+  width: 60%;
+}
+
+.logo h1 {
+  font-size: 2em;
+}
+
+header button {
+  display: flex;
+  width: 10em;
+  justify-content: center;
+  padding: .5em;
+  background-color: #4CAF50;
+  border: none;
+  color: white;
+}
+
+.checkout {
+  justify-content: center;
+  flex-direction: column;
+  align-self: flex-end;
+  display: flex;
+}
+
+i {
+  color: white;
+  font-size: 1em;
+  display: flex;
+  align-self: flex-end;
+  margin-bottom: 1em;
+}
+
+body {
+  padding-left: 5%;
+  padding-right: 5%;
+}
+
+nav {
+  background-color: #e1e7eb;
+  width: 100%;
+  padding: 1.5em;
+}
+
+nav li {
+display: inline;
+}
+
+nav a {
+ text-decoration: none;
+ font-weight: bold;
+ color: #18387d;
+ font-size: 1.2em;
+
+}
+h1 {
+  font-style: italic;
+  font-size: 1.5em;
+  color: #73c74a;
+}
+
+
+section p {
+  font-size: 1.5em;
+}
+
+
+.lead-article {
+  display: flex;
+  flex-direction: column;
+  align-self: flex-end;
+
+  margin-top: 0;
+
+}
+
+h2 a {
+  text-decoration: none;
+  color: #18387d;
+
+}
+
+h5 a {
+  text-decoration: none;
+  color: #29b1cc;
+  font-size: 1.2em;
+}
+
+aside h5 {
+  padding-bottom: 15%;
+  font-weight: bold;
+  color: #18387d;
+}
+
+h2 {
+  color: #18387d;
+  font-weight: bold;
+  font-size: 2em;
+  margin-bottom: 0;
+}
+
+h2, h3, h4, h5  {
+  font-weight: bold;
+  color: #18387d;
+}
+
+h4 {
+  font-size: 1.3em;
+}
+
+main article {
+  margin: 7%;
+  border: solid #e1e7eb .25em;
+  margin-left: 0;
+}
+
+
+main article img {
+  width: 100%;
+}
+
+
+ article button {
+   width: 90%;
+   padding: .5em;
+   display: flex;
```

Comment:
You don't need flex on this. You can center the text using text-align: center if needed.